tensorflow_datasets/core/api_utils.py

```diff
@@ -64,7 +82,7 @@ def _check_no_positional(fn, args, is_method=False, allowed=None):
     raise ValueError(_POSITIONAL_ARG_ERR_MSG % (fn.__name__, str(arg_names)))
 
 
-def _required_args(fn):
+def _required_args(fn: Callable[[], Any]) -> List[str]:
```

Comment:
Should be `Callable[..., Any]`, as the function can take arbitrary number of arguments. Same thing below

Reply:
Done